locale/pt-br/texts.po

```diff
@@ -0,0 +1,1422 @@
+msgid "Timestamp"
+msgstr "Data e Hora"
+
+#: src/screens/DecodeTx.js:91
+msgid "Decode tx"
+msgstr "Decodificar transação"
+
+#: src/screens/DecodeTx.js:91
+msgid "push"
+msgstr "forçar"
+
+#: src/screens/DecodeTx.js:91
+msgid ""
+"Write your transaction in hex value and click the button to get a human "
+"value description"
+msgstr ""
+"Escreva sua transação em hexadecimal e clique no botão para obter os detalhes"
+
+#: src/screens/DecodeTx.js:93
+msgid "Could not decode this data to a transaction"
+msgstr "Não foi possível decodificar os dados dessa transação"
+
+#: src/screens/LoadWallet.js:115
+msgid "Write the 24 words of your wallet (separated by space)."
+msgstr "Escreva as 24 palavras da sua wallet (separadas por espaço)."
+
+#: src/screens/LoadWallet.js:116
+msgid "Words separated by single space"
+msgstr "Palavras separadas por um espaço"
+
+#: src/components/BackButton.js:33 src/components/PinPasswordWrapper.js:60
+#: src/screens/LoadWallet.js:119 src/screens/NewWallet.js:164
+#: src/screens/NewWallet.js:177
+msgid "Back"
+msgstr "Voltar"
+
+#: src/screens/LoadWallet.js:120
+msgid "Import data"
+msgstr "Importar wallet"
+
+#: src/screens/LoadingAddresses.js:66
+msgid "Loading transactions..."
+msgstr "Carregando transações..."
+
+#: src/screens/LoadingAddresses.js:70
+msgid "Please wait while we load the transactions of all your addresses."
+msgstr "Por favor, aguarde enquanto nós carregamos as suas transações."
+
+#: src/screens/LoadingAddresses.js:71
+msgid ""
+"You will be automatically redirected to the wallet when we finish loading "
+"them."
+msgstr "Você será automaticamente redirecionado para a wallet quando terminar."
+
+#: src/screens/LoadingAddresses.js:72
+#, javascript-format
+msgid "<strong>Addresses found:</strong> ${ this.props.addressesFound }"
+msgstr "<strong>Endereços encontrados:</strong> ${ this.props.addressesFound }"
+
+#: src/screens/LoadingAddresses.js:73
+msgid "<strong>Transactions found:</strong> ${ this.props.transactionsFound }"
+msgstr ""
+"<strong>Transações encontradas:</strong> ${ this.props.transactionsFound }"
+
+#: src/screens/LockedWallet.js:90
+msgid "Your wallet is locked. Please write down your PIN to unlock it."
+msgstr ""
+"Sua wallet está bloqueada. Por favor, entre com o seu PIN para desbloqueá-la."
+
+#: src/components/ModalResetAllData.js:67 src/screens/LockedWallet.js:96
+#: src/screens/Settings.js:125
+msgid "Reset all data"
+msgstr "Resetar todos os dados"
+
+#: src/screens/LockedWallet.js:97
+msgid "Unlock"
+msgstr "Desbloquear"
+
+#: src/screens/NewWallet.js:154
+msgid ". A new wallet is generated by 24 words."
+msgstr ". Uma nova wallet é gerada a partir de 24 palavras."
+
+#: src/screens/NewWallet.js:155
+msgid ""
+". To have access to this wallet you must have all the words saved in the "
+"same order we will show to you."
+msgstr ""
+". Para recuperar o acesso a sua wallet, você deve salvar essas 24 palavras "
+"na mesma ordem que nós iremos te mostrar."
```

Comment:
"... na mesma ordem que **serão apresentadas**"? I know it's different from the original version but I think it sounds better.

Reply:
Way better. Thanks!